php/ui/component/class-switch-cloud.php

```diff
@@ -40,6 +42,11 @@ protected function submit_button( $struct ) {
 			'button-primary',
 		);
 
+		if ( $plugin->get_component( 'connect' )->has_connection_string_constant() ) {
+			$struct['attributes']['disabled'] = 'disabled';
+			$struct['attributes']['title']    = __( 'Defined by constant', 'cloudinary' );
+		}
```

Comment:
@pereirinha - can you change the element to a button and remove the href? The link still works.
```suggestion
			$struct['element']                = 'button';
			$struct['attributes']['href']     = null;
		}
```